src/main/scala/io/dronekit/CouchbaseStreamsWrapper.scala

```diff
@@ -292,6 +295,24 @@ class CouchbaseStreamsWrapper(host: String, bucketName: String, password: String
     observableToSource(docObservable).map(json => convertToEntity[T](json))
   }
 
+  /**
+   * Does a batch update of keys -> each update
+   * @param entities Seqence of BatchUpdateRequest to update
```

Comment:
It's weird to have a Seq[BatchUpdateRequest]. Shouldn't it be a Seq[UpdateRequest]?